cmd/debugger/main.go

```diff
@@ -304,7 +304,9 @@ func main() {
 		exitCode := 1
 		if exitErr, ok := err.(*exec.ExitError); ok {
 			exitCode = exitErr.ExitCode()
-			conslogger.Warnf("Command %s failed with exit code %d\n", quotedCmd, exitCode)
+			if debuggerSettings.Enabled {
```

Comment:
This piece of code is from the debugger. Every user command being executed is actually wrapped by the debugger. The debugger should only start interacting only if it's enabled. In this case, despite being disabled, it would add some output to failing commands. This kind of output has now been moved to its proper place (see `printError` in another file in this PR)..